packages/components/src/VirtualizedList/RowCollapsiblePanel/RowCollapsiblePanel.component.js

```diff
@@ -62,7 +55,12 @@ class RowCollapsiblePanel extends React.Component {
 						aria-label={get(rowData, 'header[0].label')}
 						style={style}
 					>
-						<PureCollapsiblePanel {...{ rowData, measure, onToggle: this.onToggle }} />
+						<CollapsiblePanel
+							onEntered={measure}
+							onExited={measure}
```

Comment:
I rather name it onExit

Reply:
I used `onEntered` and `onExited` to keep consistency with bootstrap collapse.
`onEnter` callback is called **before** component expands and `onEntered` is called **after** component has expanded. Same for `onExit` and `onExited`.
And I would like `react-virtualized` to measure the row after expand/collapse.
See https://react-bootstrap.github.io/utilities/transitions/#collapse